### DISCUSSION TITLE
Proof of Attendance Solution

### DISCUSSION THREAD

**Discussion by 1JocelynR** [community/community#138942] [GitHub Education]
Resolved --

**Comment by alx-mp4** [comment]
Hi @1JocelynR, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
